File: rust/src/models/per_unit_plan_pricing.rs
// this file is @generated
use serde::{Deserialize, Serialize};

#[derive(Clone, Debug, Default, PartialEq, Deserialize, Serialize)]
pub struct PerUnitPlanPricing {
    pub rate: rust_decimal::Decimal,
}

impl PerUnitPlanPricing {
    pub fn new(rate: rust_decimal::Decimal) -> Self {
        Self { rate }
    }
}

File: typescript/src/models/perUnitPlanPricing.ts
// this file is @generated

export interface PerUnitPlanPricing {
  rate: string;
}

export const PerUnitPlanPricingSerializer = {
  _fromJsonObject(object: any): PerUnitPlanPricing {
    return {
      rate: object["rate"],
    };
  },

  _toJsonObject(self: PerUnitPlanPricing): any {
    return {
      rate: self.rate,
    };
  },
};
